lib/jazzy/config.rb

```diff
@@ -220,6 +220,11 @@ def expand_path(path)
         SourceDeclaration::AccessControlLevel.from_human_string(acl)
       end
 
+    config_attr :lint,
+      command_line: '--[no-]lint',
+      description: "Generate linter output.",
+      default: false
```

Comment:
I was thinking the default behavior without `--dry-run` specified would be to generate the docs.